src/macos/Button.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+* Button.tsx
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* Mac-specific implementation of the cross-platform Button abstraction.
+*/
+import {Button as ButtonBase, ButtonContext} from '../native-common/Button';
+import Types = require('../common/Types');
+import React = require('react');
+import RN = require('react-native');
+
+export class Button extends ButtonBase {
```

Comment:
Can't we just put the mouse support in ButtonBase? It should be ignored on systems that don't support mice.

Reply:
I will move this part to native-common/Button, revert Mac implementation and adjust windows implementation.